extensions/wikia/TemplateClassification/services/UserTemplateClassificationService.class.php

```diff
@@ -35,12 +44,37 @@ class UserTemplateClassificationService extends TemplateClassificationService {
 	 * @throws \Swagger\Client\ApiException
 	 */
 	public function getType( $wikiId, $pageId ) {
-		$templateType = parent::getType( $wikiId, $pageId );
+		return $this->getMappedType( parent::getType( $wikiId, $pageId ) );
```

Comment:
I would even go with ```mapType``` as it's private method that does mapping for us.
getType and getMappedType may lead to question what's the difference and why one is private, what requires to read the code to make sure it's actually same.
but in general getMappedType is fine for me as well.